src/sentry/static/sentry/app/routes.jsx

```diff
@@ -1309,7 +1292,6 @@ function routes() {
             component={errorHandler(LazyLoad)}
           />
           <Route path="/organizations/:orgId/">
-            <Redirect from="/organizations/:orgId/projects/" to="/:orgId/" />
```

Comment:
I believe this was added awhile ago when we wanted to completely kill the Projects Dashboard, but it got re-added and was matched before it ever got to this Redirect -- but since we are moving the LightWeight call to take place after this route tree, this would have hit and redirected.